kubernetes/jobs/create-database.yml.j2

```diff
@@ -44,7 +44,7 @@ spec:
                   name: db-secrets
                   key:  db_user_password
         - name:             migrate-database
-          image:            {{ docker_registry }}/concent-api:latest
+          image:            {{ docker_registry }}/{{ image_prefix }}concent-api:{{ image_version }}
```

Comment:
These changes should have been made as a fixup to the previous commit that adds prefixes and versions in .yml files.